vers-one/EpubReader
# EPUB Media Overlays support and migration from Json.NET to System.Text.Json

# EPUB Media Overlays support and migration from Json.NET to System.Text.Json

This is:

Related issues: #64, #77.

## Description

This pull request:
1. Parses all SMIL files found in the EPUB book into a collection of `VersOne.Epub.Schema.Smil` objects.
2. Migrates integration tests from using *Json.NET* to *System.Text.Json*

## Testing steps

1. Open a EPUB book with media overlays and inspect the `EpubBook.Schema.MediaOverlays` property. This property should contain the list of all parsed SMIL objects that correspond to the media overlay files included in the EPUB book.
2. All integration tests should continue to pass.
